inlong-manager/manager-common/src/main/java/org/apache/inlong/manager/common/consts/SinkType.java

```diff
@@ -39,4 +45,23 @@ public class SinkType extends StreamType {
      * Details: <a href="https://www.tencentcloud.com/products/cls">CLS</a>
      */
     public static final String CLS = "CLS";
+
+    public static final Set<String> SORT_FLINK_SINK = new HashSet<>();
+    public static final Set<String> SORT_STANDALONE_SINK = new HashSet<>();
+
+    static {
+        SORT_FLINK_SINK.addAll(
```

Comment:
This method is not very flexible. If a new type is added, the code here needs to be updated. Can we achieve this functionality through annotations?

Reply:
Done